packages/firestore/src/api/timestamp.ts

```diff
@@ -19,6 +19,13 @@ import { Code, FirestoreError } from '../util/error';
 import { primitiveComparator } from '../util/misc';
 
 export class Timestamp {
+  // Midnight at the beginning of 1/1/1 is the earliest Firestore supports.
+  private static readonly MIN_SECONDS = -62135596800;
+  // This will break in the year 10,000.
```

Comment:
Nit: This constant won't break. It will continue to point to the year 10,000 even in the year 10,001. 
You can probably just mention that the value corresponds to the year 10,000.

Reply:
Haha true.  I just copied this comment from the code in its old location.  I've updated these comments to state the date to which they correspond and removed the mention of "breaking".